_episodes/03-github-pages.md

```diff
@@ -21,7 +21,114 @@ optionally runs the files through a build process, and publishes them as a websi
+>
+{: .callout}
+
+Let's start from the github repository we have created in the previous episode {{ site.project-pages }}. One important file you should have is `index.md`: it will become the homepage of your project website.
+
+In this episode, you will learn how to turn your project repository into a website.
+
+## Create gh-pages branch
+
+GitHub Pages uses a special branch in your GitHub repository to look for website content, and by default this branch is named `gh-pages`. This branch does not exist yet and the first step is to create it from the Github interface.
+
+**TODO: add figure once we agree on the name of the repo **
+
+![gh-pages branch creation](../fig/gh-pages_branch_creation.jpg)
+
+It is a convention used by Github and we will learn later on to change this default behavior.
+
+> ## GitHub interface versus command line
+> Branches can also be created from the command line but for simplicity, we will only show you how to publish a website using the GitHub interface.
+>
+{: .callout}
+
+
+
+> ## What is a branch?
+> You may have never heard about git branches and wonder what they mean. It is not so important at this stage and for now we will assume that a branch in git is similar to the branch of a tree. From now, we will make sure to update `gh-pages` branch and the default `master` branch will remain untouched.
+>
+{: .callout}
+
+
+## Configure repository as a website
+
+### View your website
+
+As mentionned above, by default, GitHub serves automatically GitHub pages from `gh-pages` branch but where can we view our website?
+
+- Click on the repository "**Settings**" as shown on the figure below:
+
+**TODO: add picture**
+
+![Repository settings](../fig/repo_settings.png)
+
+
+- Then search for "GitHub pages" to find the web address of your website and click on the shown link. Your default browser will open and show your project website:
+
+![Default project website](../fig/default_website.png)
+
+You should see the contents of the `index.md` file that we created earlier. 
+
+It can take from a few seconds to a few minutes before your website is available. It is mostly due to GitHub availability and complexity of your project website. You know it is ready when the link appears in green with a "tick" in front of the web address (as shown on the figure above).
+
+What happened behind the hood, is that GitHub converted markdown to HTML. At the top of your website, the name of your repository appears as a link to the project homepage.
+
+> ## What if my browser does not open?
+>
+> If your default web browser did not open and does not show the project website, we recommend to manually open your favourite web browser and paste the project web address.
+>
+{: .callout}
+
+
+
+> ## Add new content to your website
+>
+> Update `index.md` from the GitHub interface and commit your changes.
```

Comment:
Could we ask them to add something specific to their site, like a short about section (with header level 2) on the index page.

Reply:
- add link to the website
- add  a section *About* with a short description of the purpose of the website.

What else can we ask?